samples/qml/metrics.js

```diff
@@ -2,9 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-.import "../../dist/glean.js" as GleanInstance;
-
-const Glean = new GleanInstance.Glean.Glean("qml-sample-app", "qml")
+const Glean = new GleanInstance.Glean.Glean("qml-sample-app", "QML")
```

Comment:
Why is this changing again?

Reply:
Why indeed, probably rebase issues, there was a bunch of conflicts with my last three PRs, including  this one.